tests/e2e/create/testdata/moduleconfig/valid/local-defaultcr.yaml

```diff
@@ -0,0 +1,7 @@
+# Sample for the tests!
```

Comment:
please move to a separate folder holding defaultCRs, similar to sec-scanners-config. This then also tests if we can properly define a path relative to the config